README.md

```diff
@@ -63,178 +72,355 @@ bambuAnnotations <- prepareAnnotations(gtf.file)
 ```
-
-From *TxDb* object
+Alternatively transcript discovery can be run without thresholds, producing a GRangesList annotation object with all transcripts scored with its NDR score. Note that this means turning quant = FALSE in running bambu (refer to “Transcript discovery only” section). The annotations can be filtered by their NDR score (see example below), read count and gene read proportion between the discovery and quantification steps or used for other types of analysis. 
 
 ```rscript
-annotations <- prepareAnnotations(txdb)
+newAnnotations <- bambu(reads = test.bam, annotations = annotations, genome = fa.file, NDR = 1, quant = FALSE)
+annotations.filtered <- newAnnotations[(!is.na(mcols(newAnnotations)$txNDR) & mcols(newAnnotations)$txNDR<0.1) | is.na(mcols(newAnnotations)$txNDR)]
+se.NDR_1 <- bambu(reads = test.bam, annotations = annotations.filtered, genome = fa.file, NDR = 1, discovery = FALSE)
+
+Additionally there are other thresholds that advanced users can access through opt.discovery when running Bambu (see arguments).
 ```
+### Output
+bambu returns a SummarizedExperiment object which can be accessed as follows:
 
----
+assays(se) returns a list of transcript abundance estimates as counts or CPM
+rowRanges(se) returns a GRangesList with all annotated and newly discovered transcripts
+rowData(se) returns additional information about each transcript
 
-### Advanced Options
+Access transcript expression estimates by extracting a variable (such as counts or CPM) using assays() 
+assays(se)$counts - expression estimates
+assays(se)$CPM -sequencing depth normalized estimates
+assays(se)$fullLengthCounts - estimates of read counts mapped as full length reads for each transcript
+assays(se)$uniqueCounts - counts of reads that are uniquely mapped to each transcript
 
-**More stringent filtering thresholds imposed on potential novel transcripts**    
- 
-- Keep novel transcripts with min 5 read count in at least 1 sample: 
+For a full description of the other outputs see [Output Description](#Output-Description)
 
+The full output can be written to a file using writeBambuOutput().  Using this function will generate three files, including a .gtf file for the extended annotations, and two .txt files for the expression counts at transcript and gene levels.
 ```rscript
-bambu(reads, annotations, genome, opt.discovery = list(min.readCount = 5))
+writeBambuOutput(se, path = "./bambu/")
 ```
-
-- Keep novel transcripts with min 5 samples having at least 2 counts:
+If quant is set to FALSE i.e., only transcript discovery is performed, only the rowRanges output of the extended annotations is returned (a GRangesList object). The equivalent rowData can be accessed with mcols()
+If both quant and discovery are set to FALSE, bambu will return an intermediate object see [Storing and using preprocessed files (rcFiles)](#Storing-and-using-preprocessed-files-rcFiles)
+ 
+### Visualization
```

Comment:
In general, the figures a bit big, personally I think it looks nicer if it is smaller.